products/sle/main.pm

```diff
@@ -671,6 +671,11 @@ testapi::set_distribution(susedistribution->new());
 
 # Set serial failures
 my $serial_failures = [];
+# Detect general problems with qemu, we can simply die hard
+push @$serial_failures, {type => 'hard', message => 'poo#44432', pattern => quotemeta 'VCPU can only run on primary'};
+push @$serial_failures, {type => 'hard', message => 'poo#36559', pattern => quotemeta 'kvm run failed Device or resource busy'};
+
+push @$serial_failures, {type => 'hard', message => 'bsc#1093797', pattern => quotemeta 'Internal error: Oops: 96000006'};
```

Comment:
Whops :)

Reply:
"whops" what? this bug is being checked on in the next line, is that what you mean?